routes/weather.ts

```diff
@@ -5,7 +5,13 @@ import * as SunCalc from "suncalc";
 import * as moment from "moment-timezone";
 import * as geoTZ from "geo-tz";
 
-import { AdjustmentOptions, GeoCoordinates, TimeData, WateringData, WeatherData, WeatherProvider } from "../types";
+import { GeoCoordinates, TimeData, WateringData, WeatherData, WeatherProvider } from "../types";
+import { AdjustmentMethod, AdjustmentMethodResponse, AdjustmentOptions } from "./adjustmentMethods/AdjustmentMethod";
+import ManualAdjustmentMethod from "./adjustmentMethods/ManualAdjustmentMethod";
+import ZimmermanAdjustmentMethod from "./adjustmentMethods/ZimmermanAdjustmentMethod";
+import RainDelayAdjustmentMethod from "./adjustmentMethods/RainDelayAdjustmentMethod";
+import EToAdjustmentMethod from "./adjustmentMethods/EToAdjustmentMethod";
+
 const weatherProvider: WeatherProvider = require("./weatherProviders/" + ( process.env.WEATHER_PROVIDER || "OWM" ) ).default;
```

Comment:
weatherProvider defaults to OWM but we don't necessarily have a OWM_API_KEY defined so this could raise an error. Do we need a guard clause in `server.ts`

Reply:
It will currently fail silently and just return time data in all watering scale requests. I can look into adding some checks to inform the user if they need to setup environment variables for the server.

Reply:
Yeah, I dont think we should be hitting a public endpoint without the right authentication details. Agree not specific to this PR so defer to #57